### PR TITLE
Update BLorient796.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient796.xml [hunk 1]
```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                   <msItem xml:id="ms_i1.1">
+                     <locus from="3r"></locus>
+                     <title type="complete" ref="LIT2667Statut71Title" xml:lang="en">
+                     Orders of the Apostles, 71 titloi</title>
+                     <textLang mainLang="gez"></textLang>
+                   </msItem>
+                   <msItem xml:id="ms_i1.2">
+                     <locus from="4r"></locus>
+                     <title type="complete" xml:lang="en">Orders of the Apostles,
+                     here reckoned 81 in number.</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient796.xml:54]: This is very wrighty English :) I would adapt it, but if your policy is to change as little as possibly from the catalogue, that's fine.

File: LondonBritishLibrary/orient/BLorient796.xml [hunk 2]
```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                     <textLang mainLang="gez"></textLang>
+                   </msItem>
+                   <msItem xml:id="ms_i1.3">
+                     <locus from="30r"></locus>
+                     <title type="complete" xml:lang="en">Various forms of prayer</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient796.xml:59]: I think these are LIT4711Mastabqwe

File: LondonBritishLibrary/orient/BLorient796.xml [hunk 3]
```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                     <msItem xml:id="ms_i1.3.11">
+                       <locus from="32v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ማኅበር፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.12">
+                       <locus from="32v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ጳጳስ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                       <note>Here Mark (1737-61) is mentioned.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient796.xml:124]: My wikipedia research left me clueless on who he would be , I found no corresponding pope or metropolitan with this name and dates. @DenisNosnitsin1970 can you help?

File: LondonBritishLibrary/orient/BLorient796.xml [hunk 4]
```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+
+                 <explicit xml:lang="gez"><locus target="#134v"></locus>
+                 ዝንቱ፡ መጽሐፍ፡ ዘይሰመይ፡ ሲኖዶስ፡ ዘሠርዑ፡ አበዊነ፡ ሐዋርያት፡
+                 <subst><del></del><add place="overstrike">ዘአጥርዮ፡ በወርቁ፡
+                 <persName ref="PRS2377Bakaffa">ንጉሠ፡
+                 ነገሥት፡ በካፋ፡</persName></add></subst> በረከቶሙ፡ ትኩን፡ ምስሌሁ፡
+                 ለዓለመ፡ ዓለም፡ አሜን። ዘሰረቆ፡ <gap reason="ellipsis"></gap>
+                 </explicit>
+                 <note>According to the catalogue description the manuscript can not
+                 be so oldas the time of <persName ref="PRS2377Bakaffa"></persName>.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient796.xml:345]: cannot, old as

**Review by thea-m — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient796.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient796.xml:124]: It is not clear how Mark is referred to, as papas or liqa papas, as both can be mentioned in the supplication, if I remember well; but if there is no Ethiopian metropolitan and no Coptic Patriarch Mark fitting the years, more or less, I do not know.

**Review by thea-m — APPROVED** [review]

**Pull request merged**